student.rb

```diff
@@ -1,9 +1,13 @@
+require_relative 'classroom'
+require_relative 'book'
+require_relative 'rental'
 class Student < Person
   attr_accessor :classroom
 
   def initialize(age, name, parent_permission, classroom)
     super(age, name, parent_permission)
     @classroom = classroom
+    classroom.add_student(self)
   end
 
```

Comment:
- [ ] please ensure when setting the classroom for a student it also adds it to the classrooms' students. kindly add a method that serves as the 'belongs-to' side (a student belongs to a classroom).
```ruby
def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```